pkg/cmd/taskrun/describe.go

```diff
@@ -194,3 +198,18 @@ func hasFailed(tr *v1alpha1.TaskRun) string {
 
 	return ""
 }
+
+func sortStepStatesByStartTime(steps []v1alpha1.StepState) []v1alpha1.StepState {
+	sort.Slice(steps, func(i, j int) bool {
+		if steps[j].Terminated == nil {
```

Comment:
/hold

I'm going to look into using `Running`'s `StartedAt` so `Running` steps can also be sorted

Reply:
Resolved

/hold cancel